profile/config.go

```diff
@@ -0,0 +1,231 @@
+package profile
+
+var vmProfileConfigs map[string]*VMProfile = map[string]*VMProfile{
+	"cannon-singlethreaded-32": {
```

Comment:
Why do we need to create a hardcoded config here? @dhyaniarun1993 
I thought we could just provide file path of the profile yaml we already have

Reply:
I am embedding a default config in the tool itself so users can reference predefined settings using tags instead of providing full file paths. 

If we don't do this then optimism
- Either need to maintain the config yaml in their repository and provide it while running the analysis
- Or they need to download it from our repository and then provide it while running the analysis

Having predefined config ensures consistency and reduces the need for users to manage config files manually.